### DISCUSSION TITLE
How to get Outliner to work

### DISCUSSION THREAD

**Discussion by micwaab** [vslinko/obsidian-outliner#282] [Q&A] [answered]
Hi all,

I have been a longtime Roam and Logseq user and am thinking about transitioning to Obsidian. I'm just getting started, so bear with me. I installed the Outliner plugin, I activated it, and all the features are toggled "on." However, I can't seem to get the outliner functions to start up? How do I apply the outliner in my notes?

Thanks

**Comment by vslinko** [comment, marked as answer]
Hi @micwaab. 

To activate Outliner mode, you need to start a markdown list by printing dash `-` and space ` ` characters.
If you've done that, but Outliner functions, like moving items doesn't work - it looks like a bug.
You can send a bug report [here](https://github.com/vslinko/obsidian-outliner/issues).

**Comment by spicylentils** [comment]
I suffered with this problem too so perhaps the readme can say to put cursor anywhere within the outline, but to NOT select any text, then hit Ctrl+shift+up or Ctrl+shift+down. Otherwise, a feature request / bug fix would be to ignore the selection of text.
